### ISSUE TITLE
Use default instance names in operator documentation

### ISSUE THREAD

**Issue by guenter** [kudobuilder/frameworks#186]
The current documentation sometimes assumes default instance names, and sometimes overrides the default, so commands in the docs don't work when copied verbatim.

For example the [installation docs for Kafka](https://github.com/kudobuilder/operators/blob/master/repository/kafka/docs/v1.1/install.md#installing-the-operator) override the names with `--instance`, but Kafka tries to connect to a ZooKeeper named `zookeeper-instance`, so it never comes up.

In another example, the [docs for scaling Kafka](https://github.com/kudobuilder/operators/blob/master/repository/kafka/docs/v1.1/update.md#scaling-the-brokers) sometimes assume the default instance name, and sometimes it is overridden.

My recommendation is to use the default instance names in the docs to keep things simple, and assume default instance names in parameter such as the ZooKeeper host names. I'm happy to make a pass if people agree but wanted to post this first to see if work is already underway.

**Comment by zmalik** [comment]
yes, the documentation is outdated and was created at first place when we had random instance names. Now with the default name `operatorName-instance` we should make it simple. I will open a PR to fix it for Kafka
